sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -475,4 +485,22 @@ def get_database_account(self, **kwargs):
         result = self.client_connection.GetDatabaseAccount(**kwargs)
         if response_hook:
             response_hook(self.client_connection.last_response_headers)
+        # self.diagnostics.update_diagnostics(self.client_connection.last_response_headers, result,
+        #                                     self.client_connection.last_exceptions)
         return result
+
+    def diagnostics(self, p=False):
```

Comment:
Is there a delimiter on the Diagnostics? Imagine a file with diagnostics plus other log lines before and after the diagnostics (a customer application logging some custom data and diagnostics), and you have a machine that is trying to extract the diagnostics from these logs. We know they start with the `DIAGNOSTICS` keyword, but how do we know when they end and distinguish between them from other log lines that come after?

Which would be an example of not nicely formatted?

Reply:
I should be able to add "END DIAGNOSITCS" at the end as delimter.